.github/workflows/clang-analyzer.yml

```diff
@@ -13,6 +13,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+        with:
+          submodules: true
 
       - name: Install libmaxminddb
         run: sudo apt install clang-tools libmaxminddb-dev
```

Comment:
Do we still want to be installing libmaxminddb via apt in the various spots? (Not sure)

Reply:
Yeah, I should have removed those, but given that it sounds like we will be supporting using the shared library as well, we will probably want to test that and keep at least some of those.